console: Document purpose of importPaths (#941)
redpanda-data/documentation: docs/platform/console/features/protobuf.mdx

## GitHub repository

To provide the proto files through a GitHub repository, first place the files in a directory. The directory doesn't matter, because 

> Redpanda Console searches for all files with the `.proto` extension, up to a directory depth of five levels. To

Reviewer: ```suggestion
Redpanda Console searches for all Proto files up to a directory depth of five levels. To
```

> Redpanda Console searches for all files with the .proto extension, up to a directory depth of five levels. If you want to

Reviewer: ```suggestion
Redpanda Console searches for all files with the `.proto` extension, up to a directory depth of five levels. If you want to
```

> use multiple different import paths, you must set the `importPaths` property. A common usecase for this are repositories which use

Reviewer: ```suggestion
use multiple different import paths, set the `importPaths` property. For example, if your repository includes
```

> third-party Protobuf types, such as Google, that are in a different directory to your own types.

Reviewer: ```suggestion
third-party Protobuf types, such as Google's types, that are in a different directory to your own types, set the `importPaths` property to both directories.
```

> third-party Protobuf types (for example Google's well-known types) that live in a different directory than your own types.

Reviewer: ```suggestion
third-party Protobuf types, such as Google, that are in a different directory to your own types.
```

> third-party Protobuf types, such as Google's types, that are in a different directory to your own types, set the `importPaths` property to both directories.

Reviewer: ```suggestion
third-party Protobuf types, such as Google's types, that are in a different directory to your own types, set the `importPaths` property to the paths of both directories.
```
